### PR TITLE
107 admin views individual order

### LINKED ISSUE #107: Admin views an individual Order
As an authenticated Admin, when I visit an individual order page
      Then I can see the order's date and time.
      And I can see the purchaser's full name and address.
      And I can see, for each item on the order:
          - The item's name, which is linked to the item page.
          - Quantity in this order.
          - Price
          - Line item subtotal.
      And I can see the total for the order.
      And I can see the status for the order.

### PR DESCRIPTION
**Pull request by meyerhoferc**
* adds address attribute to a user + model tests
* adds admin view order show feature test + view
* adds .subtotal method for order_item + model test
* removes current_admin? logic from orders-controller for show action

all tests pass
closes #107

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -4,11 +4,7 @@ def index
   end
 
   def show
-    if current_admin?
-      @order = Order.find(params[:id])
-    else
-      @order = current_user.orders.find_by(params[:order_id])
-    end
+    @order = current_user.orders.find_by(params[:order_id])
```
Comment (CjMoore) [inline app/controllers/orders_controller.rb:7]: 👍

File: app/views/orders/show.html.erb [hunk 2]
```diff
@@ -13,15 +12,13 @@
       <% @order.items.each do |item| %>
       <tr>
         <td><%= link_to item.title, item_path(item) %></td>
-        <td><%= item.price %></td>
+        <td><%= number_to_currency(item.price, precision: 2) %></td>
```
Comment (CjMoore) [inline app/views/orders/show.html.erb:15]: great use of the helper.

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -9,6 +9,7 @@
     it { is_expected.to validate_presence_of(:password) }
     it { is_expected.to validate_presence_of(:first_name) }
     it { is_expected.to validate_presence_of(:last_name) }
+    it { is_expected.to validate_presence_of(:address) }
```
Comment (CjMoore) [inline spec/models/user_spec.rb:12]: Nice. 👍

**Review by CjMoore — APPROVED** [review]
Looks great. Ready for second review and merge @valerietrudell @cmacaulay

File: app/controllers/orders_controller.rb (hunk 1 above)
Comment (valerietrudell) [inline app/controllers/orders_controller.rb]: 💯

File: spec/models/user_spec.rb (hunk 3 above)
Comment (valerietrudell) [inline spec/models/user_spec.rb]: :1st_place_medal:

**Pull request merged**